scipy/spatial/distance.py

```diff
@@ -1783,7 +1783,8 @@ def pdist(X, metric='euclidean', *args, **kwargs):
         Returns a condensed distance matrix Y.  For
         each :math:`i` and :math:`j` (where :math:`i<j<m`),where m is the number
         of original observations. The metric ``dist(u=X[i], v=X[j])``
-        is computed and stored in entry ``ij``.
+        is computed and stored in entry 
+        ``m * i + j - (i + 2) * (i + 1) // 2``.
```

Comment:
I think this would be clearer with brackets
```suggestion
        ``m * i + j - ((i + 2) * (i + 1)) // 2``.
```

Reply:
Sounds good!